tests/test_quantum_info_clifford.py

```diff
@@ -17,32 +17,32 @@
 
 def construct_clifford_backend(backend):
     if isinstance(backend, TensorflowBackend):
-        with pytest.raises(NotImplementedError) as excinfo:
-            clifford_backend = CliffordBackend(backend)
-            assert (
-                str(excinfo.value)
-                == "TensorflowBackend for Clifford Simulation is not supported yet."
-            )
+        with pytest.raises(NotImplementedError):
+            clifford_backend = CliffordBackend(backend.name)
     else:
+        backend = backend.name if backend.platform is None else backend.platform
         return CliffordBackend(backend)
 
 
+def get_engine(backend):
+    return backend.platform if backend.platform is not None else backend.name
```

Comment:
As said above, this function is sensible, and it could be part of the main package itself (it is also used there)